packages/now-cli/src/commands/secrets.js

```diff
@@ -172,6 +183,22 @@ async function run({ output, token, contextName, currentTeam, ctx }) {
         console.log(`\n${out}\n`);
       }
     }
+
+    if (pagination && pagination.count === 20) {
+      const prefixedArgs = getPrefixedFlags(argv);
+      const flags = getCommandFlags(prefixedArgs, [
+        '_',
+        '--next',
+        '-N',
+        '-d',
+        '-y',
+      ]);
+      output.log(
+        `To display the next page run ${cmd(
+          `now secrets ${subcommand}${flags} --next ${pagination.next}`
+        )}`
+      );
```

Comment:
Just a little reformatting for tidiness:

```suggestion
      const nextCmd = `now secrets ${subcommand}${flags} --next ${pagination.next}`;
      output.log(`To display the next page run ${cmd(nextCmd)}`);
```